app/views/layouts/_sidebar.html.erb

```diff
@@ -146,8 +144,15 @@
             <%= sidebar_link_to "Controlling Concurrency", 'pipelines/controlling-concurrency' %>
             <%= sidebar_link_to "Branch Configuration", 'pipelines/branch-configuration' %>
             <%= sidebar_link_to "Scheduled Builds", 'pipelines/scheduled-builds' %>
-            <%= sidebar_link_to "Permissions with Teams", 'pipelines/permissions' %>
+
+          </ul>
+          <p class="Docs__nav__section-subheading">Security</p>
+          <ul class="Docs__nav__sub-nav">
+            <%= sidebar_link_to "Secrets", 'pipelines/secrets' %>
             <%= sidebar_link_to "Audit Log", 'pipelines/audit-log' %>
+            <%= sidebar_link_to "Permissions with Teams", 'pipelines/permissions' %>
+            <%= sidebar_link_to "Incoming Webhooks", 'pipelines/incoming-webhooks' %>
```

Comment:
```suggestion
            <%= sidebar_link_to "Webhooks Security", 'pipelines/incoming-webhooks' %>
```

Reply:
Mmm... I still really want to keep "Incoming in the title without making it too long 🤔